packages/firestore/test/integration/util/internal_helpers.ts

```diff
@@ -53,9 +53,10 @@ export function withTestDatastore(
   credentialsProvider: CredentialsProvider = new EmptyCredentialsProvider()
 ): Promise<void> {
   const databaseInfo = getDefaultDatabaseInfo();
-  return newConnection(databaseInfo).then(conn => {
+  return newConnection(databaseInfo).then(async conn => {
     const serializer = newSerializer(databaseInfo.databaseId);
-    const datastore = newDatastore(conn, credentialsProvider, serializer);
+    const datastore = newDatastore(credentialsProvider, serializer);
+    await datastore.start(conn);
```

Comment:
Hmm, how does this work with `await`? `start` doesn't seem to be `async`.

Reply:
"await" returns the value that is being awaited even if it is not a Promise, so it should work. That being said, we should still drop it here. Thanks for catching.